### DISCUSSION TITLE
我已经设置了权限，但还是没办法看到所有文件夹

### DISCUSSION THREAD

**Discussion by swordpp** [jlesage/docker-freefilesync#18] [General]
我的系统时群晖7.2，使用docker创建容器，设置参数里也设置了uid和gid，但还是看不到所有文件夹
![image](https://github.com/jlesage/docker-freefilesync/assets/131244293/d28fbd1a-0333-4249-bd97-c5caf076bdf6)
![image](https://github.com/jlesage/docker-freefilesync/assets/131244293/16b6469e-27a8-4454-91d1-3a7aca9cf96e)
有没有大神指导一下。谢谢

**Comment by swordpp** [comment]
![image](https://github.com/jlesage/docker-freefilesync/assets/131244293/b22ce116-c4a0-4b75-b969-1538a7db33d2)

**Reply by swordpp** [reply]
而且无法输入中文路径，也无法界面字体有乱码，设置为中文后都是乱码，貌似是缺少相应字体
![image](https://github.com/jlesage/docker-freefilesync/assets/131244293/98e4b411-e46f-4f14-8d88-f9c13865a9ee)

**Comment by 1lkei** [comment]
你要-v挂载你需要的目录，中文乱码是因为你没设置环境变量缺少字体库，作者readme里有写
